core.spec.md

```diff
@@ -205,7 +205,7 @@ When extracting the URL's `name` and `version`, processors MUST ignore any url c
 
 ```html diagram -- Ignoring meaningless parts of a URL
 <code class=anatomy>
-  <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>identity</aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#frag<aside>ignored</aside></span>
+  <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>identity</aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>/?key=val&k2=v2#frag<aside>ignored</aside></span>
```

Comment:
BTW this diagram is a bit misleading as `identity` and `name` appear to include the slash between the name and the version rather than that part being left empty. Not sure I understand HTML/CSS well enough to understand how to get that to work better, do you?

Reply:
No, I don't understand this enough either. But I'm not worried about this creating genuine confusion either since the previous section is clear in this regard, and besides, the concept of assigning meaning to subparts of an url is common enough.

Reply:
Fixed in https://github.com/apollo-specs/apollo-specs.github.io/pull/4